tests/integration/kubernetes/lib.sh

```diff
@@ -150,6 +181,23 @@ set_metadata_annotation() {
 	yq w -i --style=double "${yaml}" "${annotation_key}" "${value}"
 }
 
+# Set the command for container spec.
+#
+# Parameters:
+#	$1 - the yaml file
+#	$2 - the index of the container
+#	$N - the command values
+#
+set_container_command() {
+	local yaml="${1}"
+	local container_idx="${2}"
+	shift 2
+
+    for command_value in "$@"; do
+        yq w -i "${yaml}" "spec.containers[${container_idx}].command[+]" --tag '!!str' "${command_value}"
```

Comment:
Again, might be worth folding this line as that would make it easier to read.

I'd kinda like this line on a T-shirt though! :smile:

Reply:
Done.